modules/knative-services-connectivity.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * serverless/getting-started-knative-services.adoc
+
+[id="knative-services-connectivity_{context}"]
+= Connecting Knative Services to existing Kubernetes deployments
+
+Knative Services can call a Kubernetes deployment in any namespace, provided that there are no existing additional network barriers.
+
+A Kubernetes deployment can call a Knative Service if:
+
+* The Kubernetes deployment is in the same namespace as the target Knative Service.
+* The Kubernetes deployment is in a namespace that has manually been added to the ServiceMeshMemberRoll.
```

Comment:
```suggestion
* The Kubernetes deployment is in a namespace that has manually been added to the ServiceMeshMemberRoll in `knative-serving-ingress`.
```

Reply:
Do we need to say anything else about `knative-serving-ingress`? I'm not sure we mention this elsewhere in the docs. What is this, why is it required and how does it come to exist?